src/default_config.yaml

```diff
@@ -0,0 +1,138 @@
+# This file defines the default values used if not overriden in config.yaml
+
```

Comment:
Let’s add some warning comment here to make sure users don’t modify this file, rather point them to the correct place for their custom config.yaml.